config/connection.php

```diff
@@ -1,7 +1,7 @@
 <?php
-$servername = "127.0.0.1";
+$servername = "db";
```

Comment:
Also I suggest not changing this (unless this is specific to docker? This is just the address the mysql connection should listen to so)

Reply:
Docker-specific thing. It should be like this.